circle.yml

```diff
@@ -17,10 +17,9 @@ dependencies:
     - "project/target/streams"
   pre:
     - cd $(dirname "$0");
-    - rm -rf node_modules/ public/jspm_packages/
+    - rm -rf node_modules/
     - npm install
```

Comment:
Workflow uses teamcity now so I can remove this file all together I think. @philmcmahon ?